app/javascript/mastodon/components/short_number.tsx

```diff
@@ -0,0 +1,111 @@
+import { memo } from 'react';
+
+import { FormattedMessage, FormattedNumber } from 'react-intl';
+
+import { toShortNumber, pluralReady, DECIMAL_UNITS } from '../utils/numbers';
+
+/**
+ * Custom renderer for numbers
+ * @param {JSX.Element} displayNumber Number to display
```

Comment:
I dont think we need the full JSDoc comments here now. The params names are explicit.

If you want to keep them here, then remove the type specifier from JSDoc, they will be inferred from Typescript.

Reply:
I focused too much on keeping as much of the original code as possible.
It's remove.